scripts/test.sh

```diff
@@ -5,6 +5,7 @@ service mysql start
 
 cat example/datasets/popularize_churn.sql | mysql -uroot -proot
 cat example/datasets/popularize_iris.sql | mysql -uroot -proot
+cat example/datasets/popularize_toutiao.sql | mysql -uroot -proot
```

Comment:
Is there a way to combine `sql/testdata` and `example/datasets/*.sql`?

Reply:
Should be. Since https://github.com/sql-machine-learning/sqlflow/issues/404 is describing the same thing, I'll try to fix this in another PR.

Reply:
Sounds good!